### DISCUSSION TITLE
How to union or unionAll with jpa?

### DISCUSSION THREAD

**Discussion by SShnoodles** [querydsl/querydsl#2927] [Q&A]
Querydsl version: 4.4.0

JPAQuery or JPAQueryFactory, i cannot find union or unionAll function.
en... I didn't find any example.
teach me how to write, please. thanks.

**Comment by F43nd1r** [comment]
JPA does not support union. Either express your queries differently, or use a database specific technology, like querydsl-sql or native queries (e.g. with `JPASQLQuery`).

**Comment by jwgmeligmeyling** [comment]
You can achieve set operations in JPQL queries in Querydsl using the Blaze-Persistence integration. See my stackoverflow answer here; https://stackoverflow.com/a/62085941/2104280
